BUILDING.md

```diff
@@ -160,6 +160,22 @@ or:
 
 (before the 'make' step)
 
+### Building with CMake
+
+Install Qt and SQLite as instructed in the previous section and clone the repo to a directory (it's `sqlitebrowser` as usual).
+
+And building with these commands:
+
+```bash
+cd sqlitebrowser
+mkdir build       # You can easily make a fresh build again by removing this directory
+cd build
+cmake -DCMAKE_PREFIX_PATH=/usr/local/Cellar/qt/5.11.2_1 ..
```

Comment:
Instead of using `/usr/local/Cellar/qt/5.11.2_1`, go with `/usr/local/opt/qt`.

The `/usr/local/opt/` thing is useful, because it's a stable link to whichever version is installed.  With that, we don't have to worry about the Qt version number being embedded in the path of our instructions. :smile:

Reply:
Sure, it was my bad quick copy-n-paste from `brew` output :)